src/olympia/addons/tests/test_serializers.py

```diff
@@ -153,6 +153,24 @@ def test_basic(self):
 
         return result
 
+    def test_is_disabled(self):
+        self.addon = addon_factory(disabled_by_user=True)
+        result = self.serialize()
+
+        assert result['is_disabled'] is True
+
+    def test_is_listed(self):
+        self.addon = addon_factory(is_listed=False)
+        result = self.serialize()
+
+        assert result['is_listed'] is False
+
+    def test_is_source_public(self):
+        self.addon = addon_factory(view_source=True)
+        result = self.serialize()
+
+        assert result['is_source_public'] is True
+
```

Comment:
I love tests that check that it's the other way around before and then check that the value changed the way we expected it. But there are a few other tests that might check that so I guess it should be okay.